extensions/wikia/ContentFeeds/ContentFeeds.php

```diff
@@ -46,7 +46,7 @@ function wfContentFeedsInit() {
 	 */
 	//$wgAjaxExportList[] = '';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)